main/reference/zoe-api/zoe.md

```diff
@@ -364,8 +364,6 @@ Every **Keyword** in **give** must have a corresponding **payment**.
 const paymentKeywordRecord = harden({ Asset: quatloosPayment });
 ```
 
-<a href="offerargs"></a>
-
```

Comment:
why remove this?

Reply:
I can bring it back - does it serve any purpose?

Reply:
Yes, preserving old anchors allows any external links that use it to continue working. But you're right that there's a typo.
```suggestion
<a id="offerargs"></a>

```

Reply:
Fixed.